src/packet/key/public.rs

```diff
@@ -429,6 +431,9 @@ impl PublicKeyTrait for PubKeyInner {
             PublicParams::X25519 { .. } => {
                 unimplemented_err!("verify X25519");
```

Comment:
should this be unimplemented, or not available?

Reply:
Right, this should be "not available".

The errors around this one slowly all bitrotted in this wrong direction :sweat_smile: 
I normalized a number of errors, in fabe4c9b9567e99b907c137167563c569ffaad9a.